llarp/iwp/session.hpp

```diff
@@ -143,7 +143,23 @@ namespace llarp
         /// we are closed now
         Closed
       };
+      static std::string toString(State state);
```

Comment:
`stateToString` would be a better name

Reply:
`7f0972d` (also, sticking with UpperCamelCase)